src/mixins/nr-affiliation-mixin.ts

```diff
@@ -198,6 +199,59 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
     }
   }
 
+  /**
+ * Handle "Amalgamate Now" button.
+ * Submit an amalgamation draft depending on business type.
+ * Redirect to Dashboard.
+ * @param legalType The legal type of the IA that's being incorporated.
+ */
```

Comment:
Line these up with the first * on line 202. (See method above for example.)

Also, on line 206, the definition should be "The legal type of the amalgamated business."